packages/react-native-client/android/src/main/java/io/fishjam/reactnative/foreground/ForegroundServiceManager.kt

```diff
@@ -0,0 +1,147 @@
+package io.fishjam.reactnative.foreground
```

Comment:
**suggestion:** 
Package name is a little bit not clear. Maybe sth like foregroundService.